lib/registries/addon/overview/controller.ts

```diff
@@ -46,6 +47,10 @@ export default class Overview extends Controller {
 
     @alias('model.taskInstance.value') registration?: Registration;
 
+    constructor(...args: any[]) {
+        super(...args);
+    }
+
```

Comment:
Do you need this?

Reply:
No, not since we removed the invocation from the controller. This is now removed.